libraries/eosiolib/eosiolib.cpp

```diff
@@ -65,6 +69,76 @@ namespace eosio {
       ds >> params;
    }
 
+   using params_type = std::vector<std::pair<uint32_t, std::variant<uint16_t, uint32_t, uint64_t>>>;
```

Comment:
Can we put `params_type` in "privileged.hpp" so it could be used at other places?

Reply:
It's Ok to put in "privileged.hpp", however, I think we need an different alias name to let users know the type is only for get/set_parameters. how do you think?